etc/di.xml

```diff
@@ -4274,11 +3988,9 @@
     <!-- END PayByBank Value handlers infrastructure -->
 
     <!-- PayByBank validators infrastructure -->
-    <virtualType name="PayByBankValidatorPool" type="Magento\Payment\Gateway\Validator\ValidatorPool">
+    <virtualType name="PayByBankValidatorPool" type="BuckarooValidatorPool">
```

Comment:
not sure here, I think you override the default validators with only the IssuerValidator